daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1925,6 +1925,245 @@ class EngineTest
     }
   }
 
+  "interfaces" should {
```

Comment:
Please put this in a separate file.
`EngineTest` is already a giant mess we do not wan to grow.

Reply:
Moved!